css/main.css

```diff
@@ -325,3 +325,82 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  height: 500px;
+  margin-top: 3%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form-container {
+  /* display: flex;
+  justify-content: space-between;
+  width: 90%; */
+  margin-left: 5%;
+}
+
+.form-container p {
+  /* width: 35%; */
+  font-weight: lighter;
+  font-size: 33px;
+  margin-bottom: 33px;
+}
+
+.inputs {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.name-input {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.first-name {
+  /* display: flex; */
+  flex-direction: column;
+  width: 90%;
+}
+
+.email {
+  /* display: flex; */
+  flex-direction: column;
+  width: 100%;
+}
+
+input {
+  padding: 14px;
+  margin-bottom: 2%;
+
+  /* border-top: none;
+  border-left: none;
+  border-right: none;
+  outline: none; */
+}
+
+.btn {
+  /* width: 20%; */
```

Comment:
- [ ] It would be better if the CSS file didn't have commented code in order to make your codebase cleaner and readable :white_check_mark: :white_check_mark: :white_check_mark: